bin/mobile-center.js

```diff
@@ -36,7 +36,7 @@ function runCli() {
   runner(process.argv.slice(2))
     .then(function (result) {
       if (commandLine.failed(result)) {
-        console.log(`Command failed, ${result.errorMessage}`);
+        console.log(`Command failed: ${result.errorMessage}`);
```

Comment:
I like the `"Command failed:"` prefix but I think the error messages should begin with capital letters otherwise it looks odd when the messages contain multiple sentences:

`"Command failed: cannot run Test Cloud tests. Returning exit code 64."`